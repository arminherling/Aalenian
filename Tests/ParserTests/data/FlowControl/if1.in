define func() 
{ 
    if true
    {
        return
    }

    return
}
