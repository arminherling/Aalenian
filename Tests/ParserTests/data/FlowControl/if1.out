FunctionDefinitionStatement: {
    Name: func
    Parameters(0): {
    }
    Block(2): {
        IfStatement: {
            Condition: {
                Name: true
            }
            Block(1): {
                ReturnStatement: {
                }
            }
        }
        ReturnStatement: {
        }
    }
}
